testsuite/features/init_clients/proxy_container.feature

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) 2024 SUSE LLC
+# Licensed under the terms of the MIT license.
+#
+# The scenarios in this feature are skipped if:
+# * there is no proxy ($proxy is nil)
+# * there is no scope @scope_containerized_proxy
+#
+# Bootstrap the proxy as a Pod
+
+@skip_if_not_container_server
+@scope_containerized_proxy
+@proxy
+Feature: Setup Containerized Proxy
+  In order to use a Containerized Proxy with the server
+  As the system administrator
+  I want to register the Containerized Proxy on the server
+
+  Scenario: Log in as admin user
+    Given I am authorized for the "Admin" section
+
+# TODO: Uncomment the following scenarios when this PR is merged: https://github.com/uyuni-project/uyuni/pull/8148
+#  Scenario: Bootstrap the proxy HostOS as a Salt minion
+#    When I follow the left menu "Systems > Bootstrapping"
+#    Then I should see a "Bootstrap Minions" text
+#    When I enter the hostname of "proxy" as "hostname"
+#    And I enter "22" as "port"
+#    And I enter "root" as "user"
+#    And I enter "linux" as "password"
+#    And I select "1-PROXY-KEY-x86_64" from "activationKeys"
+#    And I click on "Bootstrap"
+#    And I wait until I see "Bootstrap process initiated." text
+#
+#  Scenario: Wait until the proxy HostOS appears
+#    When I wait until onboarding is completed for "proxy"
+
+  Scenario: Generate Containerized Proxy configuration
+    When I generate the configuration "/tmp/proxy_container_config.tar.gz" of Containerized Proxy on the server
+    And I copy the configuration "/tmp/proxy_container_config.tar.gz" of Containerized Proxy from the server to the proxy
+
+  Scenario: Run a Containerized Proxy
+    When I run "mgrpxy install podman /tmp/proxy_container_config.tar.gz" on "proxy"
+
+  Scenario: Wait until Containerized Proxy service is active
+    And I wait until "uyuni-proxy-pod" service is active on "proxy"
+    And I wait until "uyuni-proxy-httpd" service is active on "proxy"
+    And I wait until "uyuni-proxy-salt-broker" service is active on "proxy"
+    And I wait until "uyuni-proxy-squid" service is active on "proxy"
+    And I wait until "uyuni-proxy-ssh" service is active on "proxy"
+    And I wait until "uyuni-proxy-tftpd" service is active on "proxy"
+    And I wait until port "8022" is listening on "proxy"
+    # And I wait until port "8080" is listening on "proxy"
```

Comment:
Same as above. Why is this commented out?

Reply:
Same reply :)